components/DaysOfTheWeek/DaysOfTheWeekContainer.jsx

```diff
@@ -44,30 +44,22 @@ export default function DaysOfTheWeekContainer ({ setNumberOfDays, onSaveEvent,
     }
   ]);
 
-  // state to activate and disable save button (link button)
-  const [ saveButtonActive, setSaveButtonActive ] = useState(true);
-
   // on click, the buttom will change colour and the state will be updated to include selected buttons
-  const handleClick = (e) => {
-    e.preventDefault();
```

Comment:
A question: why remove `.preventDefault()` ?
is it because there is no default event set for the `onClick` event of `<DaysOfWeekButton>` ?

Reply:
Yep, it's not inside a form. I also moved the action (event) to the button itself and the container will receive a component.